pkg/client/results/testdata/mockResults/plugins/ds-errors-02/ds-errors-02.golden.json

```diff
@@ -0,0 +1,41 @@
+{
+"name": "ds-errors-02",
+"status": "failed",
+"items": [
+{
+"name": "node1",
+"status": "failed",
+"items": [
+{
+"name": "errors.json",
+"status": "failed",
+"meta": {
+"file": "errors/node1/errors.json"
+},
+"details": {
+"data": "foo",
+"error": "testerr"
+}
+}
+]
+},
+{
+"name": "node2",
+"status": "",
```

Comment:
This shouldn't be empty. #937 had seen this before and understood it to be for unknown nodes but here it is something that has a failed leaf node. Need to identify why this isnt bubbling up.

Reply:
Found the other bug; it was that `aggregateStatuses` was doing early exit on failures. That would bubble up the right values, but wasn't ensuring that every node was visited. #937 has been updated.